hack/update-codegen.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Copyright 2018 Openstorage.org
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
+CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${SCRIPT_ROOT}; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo ../code-generator)}
+
+# generate the code with:
```

Comment:
Is someone expected to modify this manually?

Reply:
No they just have to run `hack/update-codegen.sh` without any parameters. People need to run this only if they change the CRD spec.

Reply:
I meant about the `--output-base` parameter. If that isn't required please remove it

Reply:
removed